src/Stan/Observation.hs

```diff
@@ -0,0 +1,32 @@
+{- |
+Copyright: (c) 2020 Kowainik
+SPDX-License-Identifier: MPL-2.0
+Maintainer: Kowainik <xrom.xkov@gmail.com>
+
+__Observation__ — a vulnerability found in the target project by @Stan@.
+-}
+
+module Stan.Observation
+    ( Observation (..)
+
+      -- * Pretty print
+    , prettyShowObservation
+    ) where
+
+import GHC (SrcSpan)
+
+import Stan.Core.Id (Id)
+import Stan.Inspection (Inspection)
+
+
+{- | Data type to represent discovered by Stan vulnerabilities.
+-}
+data Observation = Observation
+    { observationId           :: !(Id Observation)
+    , observationInspectionId :: !(Id Inspection)
+    , observationLoc          :: !SrcSpan
```

Comment:
Nice idea to store `SrcSpan` here! Do you think in the `Analysis` data type we will store something like `Map FilePath Observation` or `[(FilePath, Observation)]` for better grouping of all observations per file?

https://github.com/kowainik/stan/blob/e662cae85056ffcc0b7b353af87b0758911cccf9/src/Stan/Analysis.hs#L24

Reply:
Good idea. Need to think more where the filepath would be more suitable to have. But I could imagine that the tuple (or separate data type) would be more convenient here.